gitops/charts/form-recognizer-3.0/values-prod.yaml

```diff
@@ -0,0 +1,89 @@
+# Default values for form-recognizer.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: "acs-form-recognizer"
+
+init:
+  replicaCount: 1
+  image:
+    #repository: alpine
+    #tag: "3.19"
+    #command: ["tail", "-f", "/dev/null"]
+    repository: image-registry.openshift-image-registry.svc:5000/0198bb-tools/acs-form-recognizer-init
+    tag: "2.2"
+    pullPolicy: IfNotPresent
+    # Overrides the image tag whose default is the chart appVersion.
+
+customTemplate:
+  replicaCount: 2
+  image:
+    repository: image-registry.openshift-image-registry.svc:5000/0198bb-tools/form-recognizer-custom-template-3.0
+    pullPolicy: IfNotPresent
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: "2022-08-31"
+  resources:
+    limits:
+      cpu: 100m
+      memory: 900Mi
+    requests:
+      cpu: 10m
+      memory: 100Mi
```

Comment:
These settings follow what you have in your spreadsheet and seem to work okay in my testing.